Appraisals

```diff
@@ -7,3 +7,8 @@ end
 
 appraise 'default' do
 end
+
+appraise 'active_record_6' do
+  gem 'activerecord', '~> 6.0.0'
+  gem 'pg_query', '~> 1.2.0'
```

Comment:
Why is the pg_query gem upgrade here as well? Shouldn't this be part of the gemspec?

Reply:
Right, I will move it to gemspec!